package.js

```diff
@@ -30,8 +30,7 @@ Package.on_use(function (api) {
     'coffeescript@1.0.4',
     'underscore',
     'velocity:core@0.4.5',
-    'velocity:shim@0.1.0',
-    'velocity:test-proxy@0.0.4'
```

Comment:
Now I know, why this dependency was there. Because otherwise the tests-proxy is not loaded by the app.
We should add the tests-proxy to the .meteor/packages file.